src/service/resources-impl.js

```diff
@@ -1493,6 +1518,8 @@ export class Resources {
       });
       this.requestsChangeSize_ = this.requestsChangeSize_.filter(
           request => request.resource != resource);
+      this.pendingBuildResources_ = this.pendingBuildResources_.filter(
```

Comment:
This is not to do with load/unload, right? Basically, if the resource has been removed from DOM - we no longer need to wait to build it.